src/Illuminate/Auth/DatabaseUserProvider.php

```diff
@@ -16,7 +16,7 @@ class DatabaseUserProvider implements UserProvider
      *
      * @var \Illuminate\Database\ConnectionInterface
      */
-    protected $conn;
+    protected $connection;
```

Comment:
Anyone extending `DatabaseUserProvider` and uses `$this->conn` will suddenly be broken if this got merged to 8.x. We should PR this to `master`.